### PR TITLE
Fix Enable/Disable of the Manual Verification Token Endpoint

### PR DESCRIPTION
**Pull request by jrmccannon**
All fixed up. Thanks @abergs

### CODE REVIEW

File: src/AdminConsole/Pages/App/Settings/Settings.cshtml.cs [hunk 1]
```diff
@@ -206,7 +206,9 @@ public async Task<IActionResult> OnPostSettingsAsync()
 
         try
         {
-            if (Convert.ToBoolean(Request.Form[ManualVerificationTokenCheckboxId].FirstOrDefault()))
+            Request.Form.TryGetValue(ManualVerificationTokenCheckboxId, out var values);
```
Comment (jonashendrickx) [inline src/AdminConsole/Pages/App/Settings/Settings.cshtml.cs:209]: Why is it not getting it from the form's model directly/

Comment (jrmccannon) [reply src/AdminConsole/Pages/App/Settings/Settings.cshtml.cs:209]: For some reason the bindings aren't working on that page. Do you see something that's missing from the previous version of the page?

**Comment by abergs** [comment]
I commited a small test - this seemed to be working for me. Can you verify?

File: src/AdminConsole/Pages/App/Settings/Settings.cshtml.cs (hunk 1 above)
Comment (jrmccannon) [reply src/AdminConsole/Pages/App/Settings/Settings.cshtml.cs]: Fixed!

**Review by abergs — APPROVED** [review]

**Pull request merged**